pygmt/src/contour.py

```diff
@@ -64,8 +64,16 @@ def contour(self, x=None, y=None, z=None, data=None, **kwargs):
         - Specify a fixed annotation interval *annot_int* or a
           single annotation level +\ *annot_int*.
     {B}
-    levels : str
-        Contour file or level(s)
+    levels : str or int
+        Specify the contour lines to generate.
+
+        - The filename of a `CPT`  file where the color boundaries will
```

Comment:
```suggestion
        - The filename of a CPT file where the color boundaries will
```

Reply:
@willschlitzer Could you also make the same fix in `grdcontour.py`?

Reply:
Done in 50fbf4e